tests/unit_tests/main_test.cpp

```diff
@@ -114,42 +114,37 @@ int main(int argc, char** argv) {
                         unique_devices.insert(dev.get_info<sycl::info::device::name>());
 #if !defined(ONEMKL_ENABLE_MKLCPU_BACKEND) &&             \
     !defined(ONEMKL_ENABLE_PORTBLAS_BACKEND_INTEL_CPU) && \
-    !defined(ONEMKL_ENABLE_PORTFFT_BACKEND) && \
-    !defined(ONEMKL_ENABLE_NETLIB_BACKEND)
+    !defined(ONEMKL_ENABLE_PORTFFT_BACKEND) && !defined(ONEMKL_ENABLE_NETLIB_BACKEND)
```

Comment:
I have a slight preference for the previous way this was formatted (each clause on its own line), but I don't think we have too many places where this happens, and it's still understandable.

Reply:
I think (almost) each clause on its own line was introduced in [PR #571](https://github.com/oneapi-src/oneMKL/pull/571/files#diff-555a5cfce78e81f8a579d1898a92141512b7d89140732263d0dc6ed8b444c221), and we haven't fixed its formatting. Actually `clang-format-9` and the old style file would give the same result.